en/docs/reference/mediators/foreach-mediator.md

```diff
@@ -1,143 +1,209 @@
+      </tr>
+   </tbody>
+</table>
+
+### Output configurations
+
 <table>
-<thead>
-<tr class="header">
-<th>Parameter Name</th>
-<th>Description</th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td id="foreachID"><strong>ForEach ID</strong></td>
-<td>If a value is entered for this parameter, it will be used as the prefix for the <code>FOREACH_ORIGINAL_MESSAGE</code> and <code>FOREACH_COUNTER</code> properties created during mediation. This is an optional parameter. It is recommended to define a ForEach ID in nested ForEach scenarios to avoid the properties mentioned from being overwritten.</td>
-</tr>
-<tr class="even">
-<td><strong>Expression</strong></td>
-<td><div class="content-wrapper">
-<p>The <a href="{{base_path}}/reference/synapse-properties/expressions">expression</a> with which different messages are derived by splitting the parent message. This expression should have matching elements based on which the splitting is carried out.</p>
-<p>You can click <strong>+ Add Namespace</strong> in the <strong>Expression Editor</strong> to add namespaces when you are providing an expression.</p>
-</div></td>
-</tr>
-<tr class="odd">
-<td><strong>Sequence</strong></td>
-<td><p>The mediation sequence that should be applied to the messages derived from the parent message. ForEach mediator is used only for transformations, thereby, you should not include <a href="{{base_path}}/reference/mediators/call-mediator">Call</a>, <a href="{{base_path}}/reference/mediators/send-mediator">Send</a>, or <a href="{{base_path}}/reference/mediators/callout-mediator">Callout</a> mediators, which are used to invoke endpoints, within this sequence.</p>
-<p>For the sequence type you can select one of the following options.</p>
-<ul>
-<li><strong>Anonymous</strong>: This allows you to define an anonymous sequence to be applied to the split messages by adding the required mediators as children of the ForEach mediator in the mediator tree.</li>
-<li><strong>Key</strong>: This allows you to pick an existing mediation sequence.</li>
-</ul></td>
-</tr>
-</tbody>
+   <thead>
+      <tr class="header">
+         <th>Parameter Name</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr class="odd">
+         <td>
+            <strong>Update Original Collection</strong>
+         </td>
+         <td>If enabled, the original list will be updated with the content. The content type should match the original collection type.</td>
+      </tr>
+      <tr class="even">
+         <td>
+            <strong>Variable Name</strong>
+         </td>
+         <td>
+            The name of the variable where the new content will be saved. This parameter is required if <strong>Update Original Collection</strong> is disabled.
+         </td>
+      </tr>
+      <tr class="odd">
+         <td>
+            <strong>Variable Type</strong>
+         </td>
+         <td>
+            The type of the variable where the new content will be saved. Supported values:
```

Comment:
```suggestion
            The type of variable where the new content will be saved. Supported values:
```

Reply:
Since we are referring to a variable defined above, leaving this as it is.